far/farlang.templ.m4

```diff
@@ -2251,12 +2251,12 @@ upd:"Highlight column separators"
 "Підсвічувати роздільники колонок"
 
 MConfigDoubleGlobalColumnSeparator
-"Удваивать глобальные разделители колонок"
-"Double global column separators"
+"Удваивать разделители полос"
+"Double stripe separators"
 "Dvojité oddělovače hlavních sloupců"
```

Comment:
Should I reset other languages to "upd:Double stripe separators"? More general, does anybody translate to other languages?

Reply:
It depends.
If the meaning is changed completely, it is better to reset to English and prepend with "upd:".
If it is more about style or wording, i.e. the outdated string still makes sense, you can leave as is and just prepend with "upd:".

Translations are provided by volunteers. Some are updated more or less regularly, some are not.